anomalib/deploy/export.py

```diff
@@ -52,21 +52,21 @@ def get_model_metadata(model: AnomalyModule) -> Dict[str, Tensor]:
 def export(
     model: AnomalyModule,
     input_size: Union[List[int], Tuple[int, int]],
-    export_mode: ExportMode,
+    output_format: ExportMode,
```

Comment:
If we change the variable name we should probably also change the enum to be consistent